fili-core/src/main/java/com/yahoo/bard/webservice/logging/RequestLog.java

```diff
@@ -308,15 +277,15 @@ public static void stopTiming(String timePhaseName) {
             LOG.warn("Tried to stop non-existent phase: {}", timePhaseName);
             return;
         }
-        timePhase.stop();
+        timePhase.close();
```

Comment:
Plus we shouldn't change this from `stop` to `close`. In my opinion, `RequestLog::stopTiming` should have the same semantics as what the try-with-resources block does (with the exception of the existence check that the try-with-resources doesn't have to worry about).